config/locales/en.yml

```diff
@@ -35,3 +35,6 @@ en:
       registration: 'Registration'
       login: 'Login'
       logout: 'Logout'
+
+  user:
+    message: "Format should be '(222)-222-2222'"
```

Comment:
use full format with country code, for example`+38`

Reply:
I have done this.